mubench.reviewsite/templates/detector.phtml

```diff
@@ -117,6 +117,40 @@ use MuBench\ReviewSite\Model\ReviewState;
                             <i class="fa fa-thumbs-o-down" aria-hidden="true" style="color:darkred;" title="Resolved: No"></i>
                             (<a href="<?= $review_url ?>/resolution"><i class="fa fa-handshake-o" aria-hidden="true" title="Resolution"></i></a>)
                         <?php endif; ?>
+                        <?php if($user && $misuse->hasReviewed($user)): ?>
+                            <?php
+                                $user_review = $misuse->getReview($user);
+                                $reviews = $misuse->getReviews();
+                            ?>
+                            <?php if($review_state == ReviewState::NEEDS_REVIEW && $user_review->hasPositiveHit()) : ?>
+                                <i class="fa fa-thumbs-o-up" aria-hidden="true" style="color:green;" title="Agreement: Yes"></i>
+                                <span class="fa-stack fa-3x" style="font-size:0.7em">
```

Comment:
This span is repeated 4 times, with the only difference being the rank. Please extract into a include file and include 4 times.